### DISCUSSION TITLE
Add statusText in responses helpers

### DISCUSSION THREAD

**Discussion by raskyer** [sergiodxa/remix-utils#158] [Ideas]
When using unauthorized or notFound, it would be cool to add the default statusText as stated here : 
https://en.wikipedia.org/wiki/List_of_HTTP_status_codes

**Comment by sergiodxa** [comment]
You can do it already:
```ts
let response = badRequest(null, { statusText: "Custom text" })
```

**Reply by raskyer** [reply]
Add the statusText by default. For now, I have custom method that calls yours by setting the default statusText
